arch/arm/src/armv8-m/mpu.h

```diff
@@ -351,6 +351,25 @@ void mpu_configure_region(uintptr_t base, size_t size,
                            MPU_RLAR_PXN); \
     } while (0)
 
+/****************************************************************************
+ * Name: mpu_priv_shmem
+ *
+ * Description:
+ *   Configure a region as privileged shared memory
+ *
+ ****************************************************************************/
+
+#define mpu_priv_shmem(base, size)                  \
+  do                                                \
+    {                                               \
+      /* The configure the region */                \
+      mpu_configure_region(base, size,              \
+                           MPU_RBAR_AP_RWNO,        \
+                           MPU_RLAR_NONCACHEABLE |  \
+                           MPU_RBAR_SH_INNER |      \
```

Comment:
I'm not sure about inner or outer shareable attribute here
https://documentation-service.arm.com/static/62d6777531ea212bb6627685?token=

Reply:
looks good.